CalibCalorimetry/HcalAlgos/src/HcalTimeSlew.cc

```diff
@@ -27,7 +27,7 @@ double HcalTimeSlew::delay(double fC, ParaSource source, BiasSetting bias, doubl
   }
   else if (source==Data || source==MC){
     if(isHPD) return std::fmin(cap,tspar0[source-1]+tspar1[source-1]*log(fC+tspar2[source-1]));
-    return std::fmin(cap,tspar0_siPM[source-1]+tspar1_siPM[source-1]*log(fC+tspar2_siPM[source-1]));
+    return cap+tspar0_siPM[source-1];
```

Comment:
yes the effective shift is 9ns.
This compensate the 6ns shift in the digi timephase (historical number from the 2010) plus a 3 ns that is missing but evident from the M2 timing fit